app/views/posts/show.html.erb

```diff
@@ -1 +1,11 @@
-<h1>post show</h1>
+<%= render partial: 'post', locals: { post: @post } %> <br>
+<%= "no comments for this post" if @post.comments.empty?%>
+<div class="comments">
+<% @post.comments.each do |comment|%>
+        <%=render comment %>
+<% end %>
+<div class="button-container">
+    <button>Pagination</a></button>
```

Comment:
Hi @geekelo,

Great job on the progress you've made with your project, especially in adding the views. Your dedication and hard work are commendable. 👏

However, there's an issue that needs to be addressed:
- [ ] [OPTIONAL] Also, kindly fix the mismatched tags in `<button>Pagination</a></button>`. Ensure that you use consistent opening and closing tags.